Tests/MapboxMapsTests/Style/StyleURITests.swift

```diff
@@ -0,0 +1,88 @@
+import XCTest
+
+#if canImport(MapboxMaps)
+@testable import MapboxMaps
+#else
+@testable import MapboxMapsStyle
+#endif
+
+//swiftlint:disable explicit_top_level_acl explicit_acl
```

Comment:
we should move these disables into Tests/.swiftlint.yml so that they apply to all the tests